src/httpclient.cc

```diff
@@ -96,21 +95,21 @@ void HttpClient::setCerts(const std::string& ca, CryptoSource ca_source, const s
   if (ca_source == kPkcs11) {
     throw std::runtime_error("Accessing CA certificate on PKCS11 devices isn't currently supported");
   } else {  // ca_source =kFile
-    boost::movelib::unique_ptr<TemporaryFile> tmp_ca_file = boost::movelib::make_unique<TemporaryFile>("tls-ca");
+    std::unique_ptr<TemporaryFile> tmp_ca_file = std::make_unique<TemporaryFile>("tls-ca");
```

Comment:
I thought `make_unique` was C++14, which I don't think we're authorized to use yet.

Reply:
Oh, now I see! You've made your own version in the meantime. That's fine with me for now, but any particular reason to prefer that over the boost version we've been using?

Reply:
I thought it is in c++11 but then I have googled implementation(https://isocpp.org/files/papers/N3656.txt) that is accepted for c++14